src/style.css

```diff
@@ -0,0 +1,195 @@
+* {
```

Comment:
## [Optional Requirement]

### Template

![template](https://cdn.dribbble.com/users/6337227/screenshots/17230478/media/e4379e3065e4ba2bea8c4767a2dbf599.png)

**[Optional]**
- [ ] You have done a great job working on the styling part but it would be very nice if you follow some great templates already on the internet to make your project look professional because it will then attract the recruiters :blush: This is an optional requirement it would be on you but it would be a great thing dear :+1: